docs/content/concepts/repositories-workspaces/workspaces.mdx

```diff
@@ -32,7 +32,7 @@ The repositories in a workspace are loaded in a different process and communicat
 
 The structure of a workspace is encoded in a yaml document. By convention is it named `workspace.yaml`. The workspace yaml encodes where to find repositories.
 
-Each entry in the workspace is referred to as a _repository location_. A repository location can include more than one repository. Each repository location is loaded in its own process that Dagster tools use an RPC protocol to communicate with. This process separation allows multiple repository locations in different environments to be loaded independently, and ensures that errors in user code can't impact Dagster system code.
+Each entry in the workspace refers to a single code location, but that code location can include more than one repository. Each code location is loaded in its own process that Dagster tools use an RPC protocol to communicate with. This process separation allows multiple code locations in different environments to be loaded independently, and ensures that errors in user code can't impact Dagster system code.
```

Comment:
What's the reasoning behind changing repository location to code location? An advantage of repository location is that they can look it up in our docs if they want to learn more.

Reply:
we can split out that part into a separate discussion - agree that if we make the change we should probably do it everywhere at once.

Reply:
My thinking was that 'repository location' feels more like (yet another) term that you need to define, but a 'code location' is more self explanatory - it's just the location of your code.

Reply:
Cool yeah, I agree that it's more self-explanatory, but also worried about introducing yet another term.